#!/usr/bin/env python
PACKAGE = "camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Kp_p", double_t, 0, "Kp_p", 3.3, 0, 15)
gen.add("Kd_p", double_t, 0, "Kd_p", 25, 0, 100)
gen.add("Ki_p", double_t, 0, "Ki_p", 0, 0, 0.005)
gen.add("Kp_r", double_t, 0, "Kp_r", 3.25, 0, 15)
gen.add("Kd_r", double_t, 0, "Kd_r", 30, 0, 100)
gen.add("Ki_r", double_t, 0, "Ki_r", 0, 0, 0.005)
gen.add("Kw_r", double_t, 0, "Kw_r", 0.25,0,0.5)
gen.add("Kw_p", double_t, 0, "Kw_p", 0.25,0,0.5)
gen.add("pulseLength",int_t,0,"pulseLength",3,0,4);
gen.add("maxAngle", double_t, 0, "maxAngle", 7.5, 0, 15)


exit(gen.generate(PACKAGE, "PulseAttitudePublisher", "PIDPulse"))
